src/Microsoft.AspNetCore.Razor.Evolution/Legacy/CSharpCodeParser.cs

```diff
@@ -31,6 +31,21 @@ internal class CSharpCodeParser : TokenizerBackedParser<CSharpTokenizer, CSharpS
                 .AddType()
                 .Build();
 
+        internal static readonly DirectiveDescriptor AddTagHelperDirectiveDescriptor =
+            DirectiveDescriptorBuilder
+                .Create(SyntaxConstants.CSharp.AddTagHelperKeyword)
+                .Build();
```

Comment:
We should be adding a string token right? That way we'd be able to just re-use the descriptors tokens instead of creating them new